### ISSUE TITLE
src/core/seat/keyboard.cpp:202:29: error: ‘wlr_multi_get_session’ was not declared in this scope

### ISSUE THREAD

**Issue by thx1111** [WayfireWM/wayfire#91]
Arch Linux
wlroots-git 0.1.r9.gc55d1542-1
```
[6/128] Compiling C++ object 'src/src@@wayfire@exe/core_seat_keyboard.cpp.o'.
FAILED: src/src@@wayfire@exe/core_seat_keyboard.cpp.o
c++ -Isrc/src@@wayfire@exe -Isrc -I../src -I. -I../ -I../src/api -I/usr/include/libdrm -I/usr/include/pixman-1 -I/usr/include/libevdev-1.0/ -I/usr/include/wayfire -I/usr/include/libpng16 -flto -fdiagnostics-color=always -pipe -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wnon-virtual-dtor -Wextra -std=c++11 -O3 -DWLR_USE_UNSTABLE -Wno-pedantic -Wno-unused-parameter -march=x86-64 -mtune=generic -O2 -fstack-protector-strong -fno-plt -D_FORTIFY_SOURCE=2  -MD -MQ 'src/src@@wayfire@exe/core_seat_keyboard.cpp.o' -MF 'src/src@@wayfire@exe/core_seat_keyboard.cpp.o.d' -o 'src/src@@wayfire@exe/core_seat_keyboard.cpp.o' -c ../src/core/seat/keyboard.cpp
../src/core/seat/keyboard.cpp: In member function ‘bool input_manager::handle_keyboard_key(uint32_t, uint32_t)’:
../src/core/seat/keyboard.cpp:202:29: error: ‘wlr_multi_get_session’ was not declared in this scope
         if (check_vt_switch(wlr_multi_get_session(core->backend), key, get_modifiers()))
                             ^~~~~~~~~~~~~~~~~~~~~
../src/core/seat/keyboard.cpp:202:29: note: suggested alternative: ‘wlr_backend_get_session’
         if (check_vt_switch(wlr_multi_get_session(core->backend), key, get_modifiers()))
                             ^~~~~~~~~~~~~~~~~~~~~
                             wlr_backend_get_session
../src/core/seat/keyboard.cpp: At global scope:
../src/core/seat/keyboard.cpp:129:13: warning: ‘bool check_vt_switch(wlr_session*, uint32_t, uint32_t)’ defined but not used [-Wunused-function]
 static bool check_vt_switch(wlr_session *session, uint32_t key, uint32_t mods)
```

Actually replacing "wlr_multi_get_session" with "wlr_backend_get_session" does seem to work.

**Comment by ammen99** [comment]
Thanks for reporting this. The problem is already fixed in the signals-v2 branch, which  I should merge quite soon, and in the meantime you can still patch wayfire as you did.

**Comment by ammen99** [comment]
Fixed because #90 is merged.
